samples/bluetooth/mesh/light_ctrl/prj.conf

```diff
@@ -48,7 +48,13 @@ CONFIG_BT_MESH_GATT_PROXY=y
 CONFIG_BT_MESH_DK_PROV=y
 
 # Bluetooth mesh models
+CONFIG_BT_MESH_SCHEDULER_SRV=y
 CONFIG_BT_MESH_LIGHT_CTRL_SRV=y
-CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_ON=3
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_ON=6
 CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_PROLONG=3
-CONFIG_BT_MESH_LIGHT_CTRL_SRV_RESUME_DELAY=30
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_RESUME_DELAY=0
```

Comment:
I added a comment about this in the previous PR, but copy paste:

I chose to change this to six, because when recalling the scene specified in toggle_scene.py (with five seconds transition time), the light will begin to fade before reaching max level. I guess this is due to the LC state machine being turned on at the beginning of the transition, then taking over before the transition completes.

The resume_delay was set to zero for testing purposes, but should probably be included if we want the LC state machine to be automatically reenabled

Reply:
When having 30s as resume_delay it gives a bit wonky behavior when scheduling periodic on/off tasks, such as in toggle_onoff.py. The timer kicks in 30 seconds after the LC state machine gets disabled, which hijacks the lights when a scheduled task is fired. The state machine gets disabled on the next scheduled action, but then comes back 30s later... Maybe there should be some functionality that prevents this from happening/postpones the resumption when other actions are fired?